### PR TITLE
[Microsoft Sqlserver] - Ingest instance and server name & query optimisation

### LINKED ISSUE #4690: [Microsoft Sqlserver] - Transaction Log query doesn't respect the database name as query parameter
Query:
`SELECT 'msdb' As 'database_name', total_log_size_in_bytes As total_log_size_bytes, used_log_space_in_bytes As used_log_space_bytes, used_log_space_in_percent As used_log_space_pct, log_space_in_bytes_since_last_backup FROM sys.dm_db_log_space_usage `msdb``

The above query doesn't respect the database name provided as an input and fetches data only for the primary database.

### PR DESCRIPTION
**Pull request by muthu-mps**
- Enhancement

## What does this PR do?

Ingests the instance name and server name.
Fine tuned the query and removed the hardcoding of database names in the response.

Pending
The Kibana visualisation support for filtering the data based on instance name needs to be done

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

Run system test and verify the instance and server name fields are getting ingested for both the datastreams.

## Related issues

- Closes #4690

### CODE REVIEW

File: packages/microsoft_sqlserver/data_stream/performance/agent/stream/stream.yml.hbs [hunk 1]
```diff
@@ -9,6 +9,8 @@ merge_results: true
 dynamic_counter_name: "{{dynamic_counter_name}}"
 driver: "mssql"
 sql_queries:
+  - query: "SELECT @@servername AS server_name, @@servicename AS instance_name;"
```
Comment (lalit-satapathy) [inline packages/microsoft_sqlserver/data_stream/performance/agent/stream/stream.yml.hbs:12]: Can we add a section to the documentation, how the server_name and instance_name populated with an example?

File: packages/microsoft_sqlserver/data_stream/transaction_log/sample_event.json [hunk 2]
```diff
@@ -1,47 +1,83 @@
 {
-    "@timestamp": "2022-06-08T10:20:14.787809Z",
-    "mssql": {
-        "metrics": {
-            "database_name": "msdb",
-            "database_id": 1,
-            "used_log_space_bytes": 41.17647171020508,
```
Comment (lalit-satapathy) [inline packages/microsoft_sqlserver/data_stream/transaction_log/sample_event.json:7]: Don't see these fields in the generated event, what happened to "used_log_space_bytes" in the sample event?

**Comment by muthu-mps** [comment]
relates to #4259

File: packages/microsoft_sqlserver/_dev/build/docs/README.md [hunk 3]
```diff
@@ -13,6 +13,11 @@ See: [SQL Server Audit page](https://docs.microsoft.com/en-us/sql/relational-dat
 
 Microsoft SQL Server has a feature that allows running multiple databases on the same host (or clustered hosts) with separate settings. Establish named instance connection by using the instance name along with the host name (Ex: `host/instance_name` or `host:named_instance_port`) to collect metrics. Details of the host configuration is provided below.
 
+### Query by Instance Name or Server Name in Kibana
+
+The data can be visualized in Kibana by filtering based on the instance name and server name. The instance name can be filtered by `mssql.metrics.instance_name` and server name by `mssql.metrics.server_name` fields.
+
```
Comment (lalit-satapathy) [inline packages/microsoft_sqlserver/_dev/build/docs/README.md:19]: Looks good.

**Review by ritalwar — APPROVED** [review]

**Review by ritalwar — APPROVED** [review]
LGTM!

**Pull request merged**